### DISCUSSION TITLE
Login error: Please confirm that you are not a robot by clicking the captcha box.

### DISCUSSION THREAD

**Discussion by cooperbaerseth** [Mathieu2301/TradingView-API#141] [Q&A] [answered]
Out of nowhere I'm getting this error when trying to grab details from my chart drawings. I've been using the API for the past few weeks and haven't seen it.. Has anyone ran into this issue or know how to get past it? I tried logging in and logging out of my account via the browser but no luck.

Thanks for the support! The API has been working great so far.

**Comment by Mathieu2301** [comment, marked as answer]
Don't login every time, use the sessionid.

**Reply by cooperbaerseth** [reply]
Sorry, I'm a bit new to the mechanics of web API calls. Would a viable solution be to schedule this login call every hour and run my get_drawings call every minute? My understanding is that the sessionid will expire at some point so exiting and restarting the program every now and then would be more reliable...

**Reply by Mathieu2301** [reply]
The sessionid never expires (or maybe after a year or more).
You shouldn't generate more than one sessionid for TW-API.
You can see the list of sessions on your account settings page.

**Reply by cooperbaerseth** [reply]
Thanks! This is working well now!!
